packages/create-docusaurus/templates/facebook/.eslintrc.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) Facebook, Inc. and its affiliates.
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Are we going to update the copyright headers for the rest of the codebase?

Reply:
Ya we should if we can. I did that with Flux [recently](https://github.com/facebook/flux/commit/f0c41931a4e206ea74cd706a9223a3e37fbb05bb)

Reply:
Shouldn't you also change the Facebook Open-Source logo on that Flux site?

This PR didn't change the logo on our own site, just the template one